packaging/centos_ol/convert2rhel.spec

```diff
@@ -96,6 +96,24 @@ install -p man/%{name}.8 %{buildroot}%{_mandir}/man8/
 %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Wed Aug 19 2020 Michal Bocek <mbocek@redhat.com> 0.12-1
+- require --enablerepo with --disable-submgr
+- fix failing conversions if gpgcheck=1 not in used custom repos
+- always logging debug info to the log file
+- unnecessary backup of kernel packages is not being performed
+- add missing python-setuptools dependency on RHEL 6 to a spec file
+- unregister from RHN Classic if in use
+- change a temporary folder path from /tmp/convert2rhel/ to /var/lib/convert2rhel
+- add the ability to specify custom RHSM url
```

Comment:
Maybe `URL`, not `url`? 
WDYT?

Reply:
Thanks, I've updated it in [the PR desc](https://github.com/oamg/convert2rhel/pull/94#issue-470117969). I'll use the `URL` in the next upstream release description. And I'll change it in the downstream spec file (dist-git).

Reply:
Or, it's easier to fix it in upstream right away: https://github.com/oamg/convert2rhel/pull/95/
@sturivny, please review.